src/Exceptions/RequestValidationException.php

```diff
@@ -37,6 +38,29 @@ public function __construct(
         $this->violations = $violations;
     }
 
+    /**
+     * This method converts symfony ConstraintViolation objects into a message array
+     * format that closely follows the laravel validation errors format.
+     *
+     * @return mixed[]
+     */
+    public function getErrors(): array
+    {
+        $errors = [];
+
+        $converter = new CamelCaseToSnakeCaseNameConverter();
+
+        foreach ($this->getViolations() as $violation) {
+            /** @var \Symfony\Component\Validator\ConstraintViolationInterface $violation */
```

Comment:
Wouldn't the return type of `getViolations` be `@return \Symfony\Component\Validator\ConstraintViolationInterface[]` to avoid needing this `@var` annotation?

Reply:
:+1: